_security-analytics/usage/rules.md

```diff
@@ -30,40 +30,119 @@ In Visual view, rule details are arranged in fields, and the links are active. S
+  
+   <img src="{{site.url}}{{site.baseurl}}/images/Security/details-rule.png" alt="The Details fields in the Create detection rule window, which include the log type, rule level, and rule status fields." width="40%">
+  
+1. In the **Detection** section, specify key-value pairs to represent the fields and their values in the log source, which will be the target for detection. These key-value pairs define the detection. You can represent the values for keys as either a single value or as a list containing multiple values. 
+   
+   To define a simple key-value pair, first place the cursor on the **Selection_1** label and replace it with a selection name that describes the key-value pair. Next, enter a preferred field from the log source as the **Key**, and then use the **Modifier** dropdown list to define how the value is handled. The following modifiers are available:
+     * `contains` – Adds wildcards on either side of the value so that it is matched anywhere in the field.
+     * `all` – In the case of a list, rather than separate values with OR, the operator becomes AND and looks for a match with all values.
+     * `endswith` – Indicates that the value is matched when it appears at the end of the field.
+     * `startswith` – Indicates that the value is matched when it appears at the beginning of the field.
+   
+   After selecting a modifier, select the **Value** radio button and then enter a value for the key in the text field that follows it.
+   
+   You can add fields for mapping a second key-value pair by selecting **Add map**. Follow the previous guidance in this step to map the key-value pair. The following image shows how this definition for two key-value pairs appears in the **Create detection rule** window.
+   
+   <img src="{{site.url}}{{site.baseurl}}/images/Security/detection1.png" alt="An example of the Detection fields." width="50%">
+   
+   To see how this definition compares to how it would be configured in the YAML file, refer to the following example:
+
+   ```yaml
+   detection:
+      selection:
+       selection_schtasks:
+         Image|endswith: \schtasks.exe
+         CommandLine|contains: '/Create '
+   ```
+   
+   To add a second selection, use the **Add selection** bar following the first selection to open another key-value pair mapping. For this selection, we'll provide values as a list. As described in the first selection, replace the **Selection_2** label with a selection name, enter a field name from the log as the key, and select a modifier from the **Modifier** dropdown list.
```

Comment:
Instead of "we'll provide", can we use "you'll provide" or "values are provided as a list"?